tools/param-name-importer/param-name-importer.csproj

```diff
@@ -12,7 +12,7 @@
     <Reference Include="System.IO.Compression" Condition="$(TargetFramework.StartsWith('net4'))" />
   </ItemGroup>
   <ItemGroup>
-    <!-- This package erroneously contains /netcoreapp3.1/SgmlReader.exe and /netcoreapp3.1/SgmlReaderDll.dll.
+    <!-- This package erroneously contains /net6.0/SgmlReader.exe and /net6.0/SgmlReaderDll.dll.
```

Comment:
This comment probably shouldn't be updated.  Also, looking at the package, it does not appear to include an `.exe` any more, so we might could delete this whole hack.